packages/template-typescript-minimal/.force_overrides

```diff
@@ -0,0 +1,11 @@
+// DISCLAIMER 
```

Comment:
Are we missing adding documentation in README.md files or similar to let users know how to use this file?

Reply:
I've create this small ticket to do this later.

https://gus.lightning.force.com/lightning/_classic/%2Fa07EE0000285F7zYAE